mempool/tx.go

```diff
@@ -14,4 +14,6 @@ type TxInfo struct {
 
 	// SenderP2PID is the actual p2p.ID of the sender, used e.g. for logging.
 	SenderP2PID p2p.ID
+
+	IsRPC bool
```

Comment:
what's this flag? meaning the transaction was received by RPC?

Reply:
I added it to be explicit about which txs come from the rpc and which do not. otherwise its very hidden, something like if the peer listed in txinfo == 0